src/services/api.js

```diff
@@ -840,3 +841,19 @@ SDKWrapper.prototype.fetchSendWyreOffers = function (fromAsset: string, toAsset:
     })
     .catch(() => ({ error: true }));
 };
+
+SDKWrapper.prototype.getUserWallets = function (accessToken: string): Promise<Accounts> {
```

Comment:
we already have api method for this: `listAccounts()`

Reply:
Thanks, nor Meggie, nor I didn't know that ;/
Didn't see this on swagger either.
I've reuse it here and will push a change shortly